build/pack.cc

```diff
@@ -450,7 +450,8 @@ static rpmRC writeHdr(FD_t fd, Header pkgh)
  */
 static rpmRC writeRPM(Package pkg, unsigned char ** pkgidp,
 		      const char *fileName, char **cookie,
-		      rpm_time_t buildTime, const char* buildHost)
+		      rpm_time_t buildTime, const char* buildHost,
+		      unsigned int rpmformat)
```

Comment:
Shouldn't that be `uint32_t` instead of `unsigned int` so that the headerPutUint32 gets the right type?

Reply:
Right, wouldn't hurt :+1: